main.tf

```diff
@@ -44,7 +63,7 @@ resource "aws_inspector_assessment_template" "assessment" {
 
 resource "aws_iam_role" "inspector_event_role" {
   count = local.scheduled_count
-  name  = "${var.name_prefix}-inspector-event-role"
+  name  = "${var.name_prefix}-inspector-event"
```

Comment:
This would be a breaking change to existing users, and a major version bump, so we'd prefer not to change names for cosmetic reasons.